libuv.sh

```diff
@@ -33,5 +33,6 @@ module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@
 module load BASE/1.0 ${GCC_TOOLCHAIN_ROOT:+GCC-Toolchain/$GCC_TOOLCHAIN_VERSION-$GCC_TOOLCHAIN_REVISION}
 # Our environment
 set LIBUV_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
+setenv LIBUV_ROOT \$LIBUV_ROOT
```

Comment:
This is not needed.

Reply:
Ciao @ktf ,
I did not try from a fresh install, but if I remove both .build_succeeded and CMakeCache.txt and compile without the line it seems to work.
I modified the PR accordingly.
Thanks,
Cheers,
Diego

Reply:
For the record, environment files do not enter the compilation part, as they are meant for runtime only.